client/src/components/Disclaimer.js

```diff
@@ -19,7 +19,9 @@ function Disclaimer(props) {
           control of how you would like to contribute your data for public health and research.
         </p>
         <div>
-          <Button className="theme-Palette-white-text" onClick={() => setModalHidden(false)}>
+          <Button className="theme-Palette-white-text"
+          //  onClick={() => setModalHidden(false)}
+           >
```

Comment:
I think it would be better to just remove this `onClick` entirely for the time being, leaving the formatting the same on one line - that way it'll just reduce the chances of a merge conflict when we implement `setModalHidden` later tonight - what do you think?

Reply:
Yes, sure! will update now

Reply:
Thanks!